packages/clay-form/src/SelectBox.tsx

```diff
@@ -129,6 +111,18 @@ interface IProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
 	spritemap?: string;
 }
 
+export const getSelectedIndexes = (
```

Comment:
Why wasn't this a `function` like `reorderDown` but a `const` instead? Just curious so I know for the future when writing these functions, also interesting to see that we do these exported functions from child to parent if we want

Reply:
This is still a function, but just using an arrow function rather than a regular `function()`. I typically I use opt to using arrow functions since they are often less lines of code and allow for implicit returns. 

I think we can typically follow these guidelines
- Use `function()` when you need to access `this` and for Object.prototype properties.
- Use => everywhere else.

Reply:
Another use case for `function()` is when you need to rely on hoisting eg.

```js
// I can use `fn()` before its definition, because it gets hoisted...
console.log(fn()); // "fn"

function fn() {
  return 'fn';
}

// Can't do that with arrow functions...
console.log(arrow()); // ReferenceError: arrow is not defined

const arrow = () => 'arrow';
```